src/applications/facility-locator/components/ProviderServiceDescription.jsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
 import { facilityTypes, ccUrgentCareLabels } from '../config';
+import { CLINIC_URGENTCARE_SERVICE } from '../constants';
+
+const urgentCareProviderNames = (query, posCodes) => {
+  if (posCodes && parseInt(posCodes, 10) === 17) {
+    return ccUrgentCareLabels.WalkIn;
+  } else if (posCodes && parseInt(posCodes, 10) === 20) {
+    return ccUrgentCareLabels.UrgentCare;
+  } else {
+    return facilityTypes.urgent_care.toUpperCase();
+  }
+};
```

Comment:
Can we add a unit test for this function?

Reply:
will do